python/pyspark/pandas/generic.py

```diff
@@ -694,8 +694,8 @@ def to_csv(
         escapechar: str, default None
             String of length 1. Character used to escape `sep` and `quotechar`
             when appropriate.
-        num_files: the number of files to be written in `path` directory when
-            this is a path.
+        num_files: the number of partitions to be written in `path` directory when
+            this is a path. This is deprecated. Use `repartition` instead.
```

Comment:
This could have different meanings. The original intention was `Use DataFrame.spark.repartition instead`, wasn't it?